tools/cli/commands/changelog.js

```diff
@@ -55,3 +57,43 @@ export async function changeloggerCli( argv ) {
 		shell: true,
 	} );
 }
+
+/** Validate arguments
+ *
+ * @param {object} argv - arguments passed to changelogger.
+ */
+function validateArgs ( argv ) {
+	// make sure we're using a valid command
+	switch ( argv.cmd ) {
+		case 'add':
+			break;
+		case 'validate':
+			break;
+		case 'version':
+			break;
+		case 'write':
+			break;
+		default:
+			throw new Error( 'Not a valid command. Use `jetpack changelog --help` for help with changelogger' );
+	}
+	// If someone doesn't pass an entire project, prompt for specifics.
+	if ( argv.project ) {
```

Comment:
Same here - this basically lets you use `jetpack changelog plugin` and it will start the prompts from choosing the specific project.